### PR TITLE
Adding variables to give users a way to set a custom hork time

### PR DESCRIPTION
**Pull request by Jizanator**
Horking time will vary on the user. Each boss will have their own hork timer.

### CODE REVIEW

**Comment by Jizanator** [comment]
Should be added on line12 of barbarian.py

**Comment by nekolr** [comment]
include your own configurations?

**Comment by Jizanator** [comment]
Sorry new to this. What does that mean? I changed 3 files one of them being the new param to include the new vars.

**Comment by nekolr** [comment]
> Sorry new to this. What does that mean? I changed 3 files one of them being the new param to include the new vars.

you can add your own configuration into the `custom.ini`

File: config/params.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [general]
 name=Botty
 monitor=0
-custom_message_hook=
+custom_message_hook=https://discord.com/api/webhooks/908525387724767322/eFuC_GX6wNmbhQxuOi5VSywTs185MKXPI8TS5QNvqmzTUsNIdxzJsqPcGvk9N2oGFTYj
```
Comment (egut125) [inline config/params.ini:5]: Don't include your personal changes to params.ini. These are defaults that should not be touched unless there is strong reason. To set your own when using Botty create a custom.ini in the same folder and overwrite you values there. 

Only include your new fields or necessary default changes.

File: config/params.ini [hunk 2]
```diff
@@ -20,46 +20,52 @@ saved_games_folder=
...
+atk_len_pindle=4.0
+atk_len_eldritch=4.0
+atk_len_shenk=5.0
 atk_len_nihlatak=4.0
+#Hork length for barbarians
+hork_time_pindle=3.0
+hork_time_eldritch=2.0
+hork_time_shenk=6.0
+hork_time_council=6.0
+hork_time_nihlatak=6.0
```
Comment (egut125) [inline config/params.ini:68]: Undo all changes in this file besides these hork_time ones

File: src/char/barbarian.py [hunk 3]
```diff
@@ -77,21 +75,19 @@ def _do_hork(self, hork_time: int):
                 Logger.warning("Turns out weapon switch just took a long time. You ever considered getting a new internet provider or to upgrade your pc?")
 
     def pre_buff(self):
-        # keyboard.send(self._char_config["weapon_switch"])
```
Comment (egut125) [inline src/char/barbarian.py:80]: These pre buff weapon switch probably shouldn't have changed with this pr. These are unrelated to hork time setting below. Remove them from this pr.

File: src/char/barbarian.py.bak [hunk 4]
```diff
@@ -0,0 +1,203 @@
+import keyboard
```
Comment (egut125) [inline src/char/barbarian.py.bak:1]: Don't commit .bak files they're unnecessary to keep in the GitHub repo.

**Review by egut125 — CHANGES_REQUESTED** [review]

**Comment by mgleed** [comment]
Closing due to lack of activity, feel free to merge master and will revisit if needed

**Pull request closed without merging**